cmd/cg-client/main.go

```diff
@@ -27,16 +27,19 @@ var (
 	// Must be a global variable in the data segment so that the build
 	// process can inject the version number on the fly when building the
 	// binary. Use only from the Usage() function.
-	Version = "No version provided"
+	Version        = "No version provided"
+	DefaultBaseURL = "https://cloud-gate.example.com"
```

Comment:
The default is in two places: here and in the Makefile. I think it's cleaner to have it only once.